integ/execute_api_fail_recovery_workflow.go

```diff
@@ -0,0 +1,16 @@
+package integ
+
+import "github.com/indeedeng/iwf-golang-sdk/iwf"
+
+type executeApiFailRecoveryWorkflow struct {
+	iwf.DefaultWorkflowType
```

Comment:
Nit: I think `iwf.WorkflowDefaults` will be enough.

Reply:
I have updated this part. Please kindly review the PR again 👍